geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_Makefile_tmpl

```diff
@@ -730,19 +730,19 @@ $(CSS_XAPI_OUTPUT): \
 
 docker-compose-build.yaml: /build/requirements.timestamp
 
-.PHONY: config-docker
-config-docker: $(shell docker-required --path .) \
+.PHONY: docker-build-config
+docker-build-config: $(shell docker-required --path .) \
```

Comment:
what is the config image for ?

Reply:
To configure the print, mapserver, mapcache, tinyows, ...